test/suite/test_backup17.py

```diff
@@ -52,60 +52,19 @@ class test_backup17(backup_base):
 
     nops = 1000
 
-    def take_incr_backup(self, id, consolidate):
-        # Open the backup data source for incremental backup.
-        buf = 'incremental=(src_id="ID' +  str(id - 1) + '",this_id="ID' + str(id) + '"'
-        if consolidate:
-            buf += ',consolidate=true'
-        buf += ')'
-        bkup_c = self.session.open_cursor('backup:', None, buf)
-        lens = []
+    #
+    # With a file length list, and given consolidate option, check
+    # that we get the appropriate lengths from incremental backups
+    #
+    def check_consolidate_sizes(self, file_lens, consolidate):
```

Comment:
Hey @sueloverso  for this function, could there be a better way to explain what the function is doing?

Reply:
Well, what it says is accurate. Maybe more detail would be helpful. Here's detail:
When the consolidate option is used, we expect the incremental backup to collapse adjacent blocks and return us lengths that exceed the granularity setting. Verify we see multiple blocks. If consolidate is not used, no block lengths should ever be greater than the granularity setting.

Reply:
thanks ill add in that comment